tests/ctests/test_netplan_parser.c

```diff
@@ -418,7 +418,7 @@ test_parse_utf8_characters(__unused void** state)
 
     netplan_state_dump_yaml(np_state, fd, NULL);
 
-    size = lseek(fd, 0, SEEK_CUR) + 1;
+    size = (size_t)lseek(fd, 0, SEEK_CUR) + 1;
```

Comment:
note (non-blocking): `lseek()` returns `off_t` which is a signed int, that should easily fit into a `size_t` type.

Reply:
`off_t` can be a `long int`, see below:

```
../tests/ctests/test_netplan_parser.c:421:12: error: conversion to ‘size_t’ {aka ‘long unsigned int’} from ‘__off64_t’ {aka ‘long int’} may change the sign of the result [-Werror=sign-conversion]
```

In this particular case it's lseeking to zero so it's safe do to the cast.

Reply:
Unless of course lseek returns an error (-1)